### ISSUE TITLE
[BUG] myloader compatibility with pre-0.11.2 mydumper output

### ISSUE THREAD

**Issue by Xon** [maxbube/mydumper#429]
**Describe the bug**
After #399, `mydumper` has change it's exported file structure. However when given a backup created by mydumper v0.10.7-2 the restore fails with a bad error message;
"It was not possible to process file: sbforums.xf_activity_summary_definition-schema.sql"

The backup successfully restores using 0.10.7-2.

**To Reproduce**
Command executed (using v0.10.7-2):
`mydumper -c -e --complete-insert -B sbforums --outputdir sbforums`
Command executed (using v0.11.1-3):
`myloader -B sbforums-test -s sbforums -d sbforums`

**Expected behavior**
myloader to restore backups created before 0.11.x

I've got a number of backup created from older versions going back years, and I would like to not be needed to pin to the older version of myloader to be able to restore them.

**Backup**
`ls -l`
```
total 23756132
-rw-r--r-- 1 xon users          75 Oct  8 08:47 metadata
-rw-r--r-- 1 xon users         112 Oct  8 08:16 sbforums-schema-create.sql.gz
-rw-r--r-- 1 xon users           1 Oct  8 08:47 sbforums.xf_activity_summary_definition.metadata
-rw-r--r-- 1 xon users         254 Oct  8 08:19 sbforums.xf_activity_summary_definition-schema.sql.gz
-rw-r--r-- 1 xon users         213 Oct  8 08:16 sbforums.xf_activity_summary_definition.sql.gz
-rw-r--r-- 1 xon users           1 Oct  8 08:47 sbforums.xf_activity_summary_section.metadata
...
```

**Environment (please complete the following information):**
 - OS version: Ubuntu 18.04
 - MyDumper version: v0.11.1-3

**Comment by Xon** [comment]
Duplicate of #418 

It would be useful if there was some provided way to migrate a backup to the new filename format or to flag to look at the old filename format.

This sort of change is very painful for a backup software without any migration path.

**Comment by davidducos** [comment]
Hi @Xon,
In future releases we are going to take this into account. However, take into account that until we don't release v1.0.1, changes to the format and major changes on the engine will be made. 
Another thing to consider is that backups should be restored with the same version that has been taken.
